src/olympia/blocklist/management/commands/import_blocklist.py

```diff
@@ -14,20 +16,40 @@
 class Command(BaseCommand):
     help = ('Populate AMO blocklist by importing v2 JSON blocklist from kinto')
 
-    KINTO_JSON_BLOCKLIST_URL = 'https://firefox.settings.services.mozilla.com/v1/buckets/blocklists/collections/addons/records'  # noqa
-
     def handle(self, *args, **options):
-        log.debug(
-            'Downloading blocklist from %s', self.KINTO_JSON_BLOCKLIST_URL)
-        response = requests.get(self.KINTO_JSON_BLOCKLIST_URL)
-
+        LEGACY_BLOCKLIST_URL = (
+            settings.REMOTE_SETTINGS_API_URL +
+            'buckets/blocklists/collections/addons/records')
+        log.debug('Downloading blocklist from %s', LEGACY_BLOCKLIST_URL)
+        response = requests.get(LEGACY_BLOCKLIST_URL)
+
+        data = response.json()['data']
```

Comment:
I feel like this isn't really safe.. I ran the command locally and 💥

```
{"Timestamp": 1588694965152747264, "Type": "z.amo.blocklist", "Logger": "http_app_addons", "Hostname": "70c3b12064a3", "EnvVersion": "2.0", "Severity": 7, "Pid": 51, "Fields": {"uid": "<anon>", "remoteAddressChain": "", "msg": "Downloading blocklist from https://kinto.dev.mozaws.net/v1/buckets/blocklists/collections/addons/records"}, "severity": 100}
Traceback (most recent call last):
  File "./manage.py", line 11, in <module>
    execute_from_command_line(sys.argv)
  File "/deps/lib/python3.6/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/deps/lib/python3.6/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/deps/lib/python3.6/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/deps/lib/python3.6/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/code/src/olympia/blocklist/management/commands/import_blocklist.py", line 26, in handle
    data = response.json()['data']
KeyError: 'data'
```

Reply:
https://github.com/mozilla/addons-server/pull/14142/commits/b145f51565b37e8bad706c535466af145dcfbb4b